### ISSUE TITLE
Autocomplete always appears after semicolon

### ISSUE THREAD

**Issue by pandasauce** [microsoft/TypeScript-Sublime-Plugin#776]
Ubuntu Linux 20.04, Sublime Text 4107

The autocompletion popup always appears at the end of the line after entering a semicolon, resulting in triggering an autocomplete insertion on almost every new line (pressing Enter) that follows a semicolon.

**Comment by sicklybeans** [comment]
Yep, I'm having this exact same problem. I think I *might* have found a fix. The default package settings includes the line:

`
"auto_complete_triggers" : [ {"selector": "source.ts", "characters": "."} ],
`

Change this to:

`
"auto_complete_triggers" : [ {"selector": "source.ts", "characters": "\\."} ],
`

i.e. replace `"."` with `"\\."`. I think the problem is that `"."` is interpreted as a regular expression where `.` matches all characters. If you escape it appropriately, it seems to only match a literal period. That's just a guess though.

**Comment by sicklybeans** [comment]
I'm having the exact same problem. I think I have to disable this (very useful) plugin for now because this bug renders it unusable : (

**Comment by gas0324** [comment]
我也希望可以禁用输入分号，逗号之后的自动弹出窗口

**Comment by alexkuz** [comment]
Same story with backquote:
```ts
const Component = styled.div`
                             ^ popup triggers here
```

**Comment by gas0324** [comment]
Microsoft has abandoned maintenance
